ai/docs/Vicuna Model Deployment and Training Documentation.md

```diff
@@ -0,0 +1,56 @@
+# Vicuna Model Deployment and Training Documentation
+## Introduction
+
+[Vicuna model](https://lmsys.org/blog/2023-03-30-vicuna/) is an open-source chatbot model trained by fine-tuning LLaMA on user-shared conversations collected from ShareGPT. It has superior performance compared to LLaMA and Alpaca, even approaching the performance of ChatGPT, evaluted by GPT-4 and with comparably low training costs. The code and model parameters [in open access with github](https://github.com/lm-sys/FastChat).
+
+This document will focus on deploying and fine-tuning Vicuna model. We re-encapsulate the FastChat API for Vicuna, make it easier for personal use. Currently, it works well with the lastest version of vicuna-7b-v1.5-16k and FastChat(v0.2.33). Please check the [version compatibility between Vicuna and FastChat](https://github.com/lm-sys/FastChat/blob/main/docs/vicuna_weights_version.md) if there's any issues of version.
+
+Before use, you should have **GPU** in your system, with **enough GPU memory** [**(14GB for Vicuna-7B and 28GB for Vicuna-13B)**](https://github.com/lm-sys/FastChat#single-gpu). Also, make sure that **docker, docker-compose, nvidia-docker, jq** are installed in your Linux system.
```

Comment:
@zgy0817 I have a question here: Do we have to preinstall docker for nvidia-docker? Is it possible to directly install nvidia-docker without docker installed?

Reply:
Docker installed is a prerequisite for installing nvidia-docker, according to this link:
https://blog.csdn.net/qq_41776453/article/details/129794608